### PR TITLE
Add slo targets to SLO dashboard

### PR DESCRIPTION
**Pull request by fiunchinho**
Adding the SLO targets to the SLO dashboard like shown here

image

### CODE REVIEW

**Review by QuentinBisson — APPROVED** [review]

**Review by kopiczko — APPROVED** [review]

File: dashboards/service-level.jsonnet [hunk 1]
```diff
@@ -163,3 +163,26 @@ stdlib.dashboard(
...
+    { color: '#EAB839', value: 90 },
+    { color: 'green', value: 99 },
+  ])
+  .addTarget(
+    grafana.prometheus.target(
+      '1- min(slo_target) by (service)',
+      legendFormat='',
+    )
+  ),
+  gridPos={x:0, y: 27, w: 24, h: 5},
```
Comment (kopiczko) [inline dashboards/service-level.jsonnet:187]: Why is `y: 27`?

Comment (fiunchinho) [reply dashboards/service-level.jsonnet:187]: I just copied the values from Antonia's dashboard, I'm not sure what this actually does.

Comment (anvddriesch) [reply dashboards/service-level.jsonnet:187]: To be honrest I have no idea either. I just created this in the GUI.

**Review by anvddriesch — APPROVED** [review]
It's beautiful!

**Pull request merged**